bids/layout/layout.py

```diff
@@ -465,6 +465,13 @@ def get(self, return_type='object', target=None, extensions=None,
 
         Returns:
             A list of BIDSFiles (default) or strings (see return_type).
+
+        Notes:
+            As of pybids 0.7.0 some keywords have been changed. Namely: 'type'
+            becomes 'suffix', 'modality' becomes 'datatype', 'acq' is removed
+            and 'mod' becomes 'modality'. Using the wrong version could result
+            in get() silently returning wrong or no results. See the changelog
+            for more details.
```

Comment:
thank you for this, we are in the same boat here -- see #402

Reply:
FWIW, also fixing the tutorial in #403

Reply:
I am really happy if this helped someone!